template_package/carma_package

```diff
@@ -76,7 +143,14 @@ for f in ./**; do
         continue
     fi
 
+    # Here the files will be renamed. 
```

Comment:
I'm a little confused by what this block is doing, but perhaps that's not relevant for this PR provided the below rename stuff works. Just seems like a lot of hoops to jump through to rename a folder?

Reply:
In order for the cmake to work the cpp file is specified. These lines ensure that all node/plugin types result in only one file name for usage in the cmake